index.html

```diff
@@ -173,10 +172,40 @@ <h3>Multi-Post Stories Gain+Glory</h3>
                                 <li><a href="#">JavaScript</a></li>
                                 <li><a href="#">HTML</a></li>
                             </ul>
-                            <button class="button">See Project</button>
+                            <button class="button" id="btnPopup6">See Project</button>
+                        </div>
+                    </div>
+                </section>
+                <section class="mobile-popup-section" id="mobile-popup-section">
+                    <div class="mobile-popup" id="mobile-popup">
+                        <img id="img-pop" src="" alt="">
+                        <li class="close-popup"><i id="xbut" class="fa-solid fa-xmark"></i></li>
+                        <div>
+                            <h3 id="h3-popup" class="h3-mobile">Keeping track of hundreds of components</h3>
+                        </div>
+                        <div class="main-popup">
+                            <h3 id="h3-popup" class="h3-desktop">Keeping track of hundreds of components</h3>
+                            <div class="btn-container-desktop">
+                                <a class="button btn-popup" id="btn-1">See live<i class="fa-solid fa-arrow-up-right-from-square"></i></a>
+                                <a class="button btn-popup" id="btn-2" href="">See source<i class="fa-brands fa-github"></i></a>
+                            </div>
+                        </div>
+                        <div class="details-popup">
+                            <ul>
+                                <li><a id="badge1" href="#">Ruby on rails</a></li>
+                                <li><a id="badge2" href="#">CSS</a></li>
+                                <li><a id="badge3" href="#">JavaScript</a></li>
+                                <li><a id="badge4" href="#">HTML</a></li>
+                            </ul>
+                        </div>
+                        <p id="p-pop">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea</p>
+                        <div class="btn-container">
+                            <a class="button btn-popup" id="btn-1">See live<i class="fa-solid fa-arrow-up-right-from-square"></i></a>
```

Comment:
- [ ] I think everything is related to the pop up you should create it by javascript to kindly remove the code from here and generate everything with js since it is one of the requirements 🙏🏻🔥😊💯